static/app/utils/profiling/hooks/useVirtualizedTree/useVirtualizedTree.tsx

```diff
@@ -146,16 +226,30 @@ export function useVirtualizedTree<T extends TreeLike>(
       newTree.sort(props.sortFunction);
     }
 
+    const tabIndex = findCarryOverIndex(
+      latestStateRef.current.tabIndexKey
+        ? tree.flattened[latestStateRef.current.tabIndexKey]
+        : null,
+      newTree
+    );
+
+    if (tabIndex) {
+      updateGhostRow({
+        ref: clickedGhostRowRef,
+        tabIndexKey: tabIndex,
+        rowHeight: props.rowHeight,
+        scrollTop: latestStateRef.current.scrollTop,
+        backgroundColor: theme.blue300,
```

Comment:
I think we have some predefined colors in the theme for hover and active states. Perhaps we should try to use them here.

Reply:
Gave it a shot, but it's wayy to dark and purple looking for this use-case :/